client/src/main/scala/com.gu.contentapi.client/ContentApiClient.scala

```diff
@@ -68,11 +69,9 @@ trait ContentApiClient {
     * @param query the query
     * @return a future resolving to an unmarshalled response
     */
-  def getResponse[Q <: ContentApiQuery](query: Q)(
-    implicit 
-    decoder: Decoder[Q],
```

Comment:
There's a good trick going on here that we carry on using - Scala searches for instances of implicit classes in their companion objects - we want an implicit instance of `Decoder`, so we keep on putting the implicit instances in the `Decoder` companion object.

https://stackoverflow.com/questions/5598085/where-does-scala-look-for-implicits#:~:text=bring%20in%20implicits.-,Companion%20Objects%20of%20a%20Type,-There%20are%20two